src/HTTPBackend.js

```diff
@@ -62,14 +62,16 @@ export default function(options) {
 		delete headers.parameters;
 		delete headers.queryParameters;
 
-		return {
-			header: { ...headers, statusCode: 400 },
-			body: {
-				errors: {
-					message: err.message
-				}
+		/* Status code 412 means "precondition failed", which is sort of what happened
+		 * when the fetch api itself throws an Error.
+		 */
+		err.header = {...headers, statusCode: 412};
+		err.body = {
+			errors: {
+				message: "Unable to perform http request -- fetch threw a client side error"
```

Comment:
In Canopy's case, this message will never get logged to sentry because err.message (which is specific to what actually happened) is what will get logged.